packages/@coorpacademy-components/src/molecule/setup-cohort-item/test/fixtures/create-new.js

```diff
@@ -0,0 +1,102 @@
+export default {
+  props: {
+    fields: [
+      {
+        type: 'text',
+        title: 'Name',
+        value: '',
+        required: true,
+        autoFocus: true
+      },
+      {
+        type: 'splitScreen',
+        conditions: {
```

Comment:
it's strange , `conditions` is not an array ....

Reply:
I would expect a condition array. The name is not good. I propose providerCondition

Reply:
It's Done